### ISSUE TITLE
problem in running ww3_shel of ww3 v6.07

### ISSUE THREAD

**Issue by mubar5656** [NOAA-EMC/WW3#477]
i have compile ww3 v6.07 and trying to run ww3_shel, but i always gets the error below

The file with input data is FORMATTED.
At line 1117 of file ww3_shel.F90 (unit = 10, file = './ww3_shel.inp')
Fortran runtime error: Bad integer for item 1 in list input

i realized the time formats of the data in the ww3_shel is not being read/working when i run it. 
i have ww3 v5.16 and it works perfectly. that was what i used to realized the time isn't working in the v6.07.
any help?

**Comment by ukmo-ccbunney** [comment]
@mubar5656  Can you attach/paste an example of your input file?

**Comment by mubar5656** [comment]
am trying to attach the imput file but the system doesn't support .nc files

**Comment by ukmo-ccbunney** [comment]
I meant your `ww3_shel.inp` file

**Comment by mubar5656** [comment]
i have to convert it to txt
[ww3_shel.txt](https://github.com/NOAA-EMC/WW3/files/7252576/ww3_shel.txt)

**Comment by mubar5656** [comment]
my email mubar56@yahoo.com if you will to get the .inp file

**Comment by ukmo-ccbunney** [comment]
What switches are you compiling with?

**Comment by mubar5656** [comment]
i used NCEP_st2
[switch.txt](https://github.com/NOAA-EMC/WW3/files/7257049/switch.txt)

**Comment by ukmo-ccbunney** [comment]
Hi @mubar5656 - I have reproduced your error and it is due to an old bug the code (see #13). Essentially, the parameters for the "Separated wave field data (Type 6)" section in `ww3_shel.inp` is incorrectly read in twice.
This was corrected as a hotfix to the master branch here: edb73a36d25250b5f97cfeeff0f3f956f55cb751

You have two options at this point:
  1. If you want to keep using release v6.07. manually apply the fix to your code.  Comment out or delete line 1303 in `ww3_shel.ftn`:
```fortran
!/DEBUGINIT              write(740+IAPROC,*), 'Before reading IPRT'
!/DEBUGINIT              write(740+IAPROC,*), 'Before read 2002, case 10'
              READ (NDSI,*) IPRT, PRTFRM    ! <=== Delete this line
!/DEBUGINIT              write(740+IAPROC,*), ' After read 2002, case 10'
```

  2. checkout the HEAD of the `master` branch and use that (or `master` from edb73a36d25250b5f97cfeeff0f3f956f55cb751 or later)
